tests/manage/pv_services/test_ceph_daemon_kill_during_resource_deletion.py

```diff
@@ -253,7 +253,7 @@ def test_ceph_daemon_kill_during_pod_pvc_deletion(
         # Verify pods are deleted
         for pod_obj in self.pod_objs:
             assert pod_obj.ocp.wait_for_delete(
-                pod_obj.name, 180
+                pod_obj.name, 360
```

Comment:
Are these changes being done for this issue #4541 . Please refer to @petr-balogh's [comment](https://github.com/red-hat-storage/ocs-ci/issues/4541#issuecomment-877038583) to increase timeout only for IBM Cloud.
I have also raised a bug [1980873](https://bugzilla.redhat.com/show_bug.cgi?id=1980873) for the issue.

Reply:
Added the condition for ibmcloud platform